Podfile.lock

```diff
@@ -1,26 +1,34 @@
 PODS:
-  - CSwiftV (0.0.7)
-  - PathKit (0.8.0)
-  - Realm (3.0.0-beta.4):
-    - Realm/Headers (= 3.0.0-beta.4)
-  - Realm/Headers (3.0.0-beta.4)
-  - SSZipArchive (2.0.7)
+  - CSwiftV (0.0.9)
+  - PathKit (1.0.0)
+  - Realm (5.4.4):
+    - Realm/Headers (= 5.4.4)
+  - Realm/Headers (5.4.4)
```

Comment:
It will fail to build ImportObjectSchema.swift as ObjectId and Decimal128 were added in v10.
Please update Realm pod to the latest version.

Reply:
I've committed the updated cocoapods.